WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginListFragment.java

```diff
@@ -74,30 +59,100 @@ public static PluginListFragment newInstance(@NonNull SiteModel site, @NonNull P
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        ((WordPress) getActivity().getApplication()).component().inject(this);
-
-        mSite = (SiteModel) getArguments().getSerializable(WordPress.SITE);
         mListType = (PluginListType) getArguments().getSerializable(ARG_LIST_TYPE);
+    }
 
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
         // this enables us to clear the search icon in onCreateOptionsMenu when the list isn't showing search results
         setHasOptionsMenu(mListType != PluginListType.SEARCH);
 
-        List<SitePluginModel> sitePlugins = mPluginStore.getSitePlugins(mSite);
-        for (SitePluginModel plugin: sitePlugins) {
-            mSitePluginsMap.put(plugin.getSlug(), plugin);
-        }
-
-        if (savedInstanceState != null) {
-            mCanLoadMore = savedInstanceState.getBoolean(KEY_CAN_LOAD_MORE);
-            mIsLoadingMore = savedInstanceState.getBoolean(KEY_IS_LOADING_MORE);
-        }
+        // Use the same view model as the PluginBrowserActivity
+        mViewModel = ViewModelProviders.of(getActivity()).get(PluginBrowserViewModel.class);
+        setupObservers();
     }
 
-    @Override
-    public void onSaveInstanceState(Bundle outState) {
-        super.onSaveInstanceState(outState);
-        outState.putBoolean(KEY_CAN_LOAD_MORE, mCanLoadMore);
-        outState.putBoolean(KEY_IS_LOADING_MORE, mIsLoadingMore);
+    private void setupObservers() {
+        mViewModel.getSitePlugins().observe(this, new Observer<List<SitePluginModel>>() {
+            @Override
+            public void onChanged(@Nullable final List<SitePluginModel> sitePlugins) {
+                if (mListType == PluginListType.SITE) {
```

Comment:
This is another private member reference outside of the private scope creating a synthetic accessor method adding to our app method count.  We can make `mListType` protected or package-private to avoid that.

Reply:
All synthetic accessor method issues should be fixed in 8244bd59d148e7e560c82951daded148e249208f.